.github/workflows/main-workflow.yml

```diff
@@ -16,14 +16,14 @@ jobs:
     - name: tshirts fail
       run: |
         g++ -Wall -Wextra -Werror tshirts.cpp -o tshirts.out
-        bash .github/workflows/expectfail.sh ./tshirts.out

```

Comment:
Removing this entire line means the binary is not executed at all. So this workflow passes just by compiling.
Retain just the `./tshirts.out` to run it